express-notes-api/index.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+
+const jsonMethod = express.json();
+app.use(jsonMethod);
+
+const data = require('./data.json');
+const dataArray = [];
+
+app.get('/api/notes', (req, res) => {
+  for (const key in data.notes) {
+    if (!data.notes[key]) {
+      res.status(200).json(dataArray);
+    } else {
+      dataArray.push(data.notes[key]);
+    }
+  }
+  res.status(200).json(dataArray);
+});
```

Comment:
Performing a `GET /api/notes` multiple times in a row should always produce the same output.